swagger/src/main/scala/org/http4s/rho/swagger/models.scala

```diff
@@ -3,6 +3,7 @@ package org.http4s.rho.swagger
 import io.swagger.{models => jm}
 
 import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
```

Comment:
Having both `Converters` and `Conversions` together feels very weird... perhaps we could use only `Converters`?